resource/configmap/create.go

```diff
@@ -0,0 +1,60 @@
+package configmap
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/giantswarm/microerror"
+	v1 "k8s.io/api/core/v1"
```

Comment:
```suggestion
	corev1 "k8s.io/api/core/v1"
```

Minor but I prefer this alias.

Reply:
Done. `corev1` import is used now.